Neos.ContentRepository.Core/Classes/Projection/ContentGraph/NodeAggregateIdPath.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Neos\ContentRepository\Core\Projection\ContentGraph;
```

Comment:
i think this belongs more to `\Neos\Neos\Presentation` where there is also a `VisualNodePath`

Reply:
Sure, wasn't sure where to put this. So I put it next to AbsoluteNodePath

Reply:
Done